src/components/pipelines-overview/list-pages/PipelineRunsListPage.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Card, CardBody, Grid, GridItem } from '@patternfly/react-core';
+import { mainDataType } from '../utils';
+import PipelineRunsForRepositoriesList from './PipelineRunsForRepositoriesList';
+import PipelineRunsForPipelinesList from './PipelineRunsForPipelinesList';
+import StatusDropdown from '../StatusDropdown';
+import SearchInputField from '../SearchInput';
+
+type PipelineRunsForPipelinesListProps = {
+  mainData?: mainDataType[];
+};
+
+const PipelineRunsListPage: React.FC<PipelineRunsForPipelinesListProps> = ({ mainData }) => {
+  const { t } = useTranslation('plugin__pipeline-console-plugin');
+  const [pageFlag, setPageFlag] = React.useState(1);
+  const handlePageChange = (pageNumber: number) => {
+    setPageFlag(pageNumber);
+  };
+  return (
+    <Card className="pipeline-overview__pipelinerun-status-card">
+      <CardBody className="pipeline-overview__pipelinerun-status-card__title">
+        <Grid hasGutter className="pipeline-overview__listpage__grid">
+          <GridItem span={9} className="pipeline-overview__listpage__griditem">
+            <StatusDropdown />
+            <SearchInputField pageFlag={pageFlag} />
+          </GridItem>
+          <GridItem span={3}>
+            <Button
+              id="repository-button"
+              variant="tertiary"
+              className="pipeline-overview__listpage__button"
+              onClick={() => handlePageChange(2)}
+              isActive={pageFlag === 2}
+            >
+              {t('Per repository')}
+            </Button>{' '}
+            <Button
+              id="pipeline-button"
+              variant="tertiary"
+              className="pipeline-overview__listpage__button"
+              onClick={() => handlePageChange(1)}
+              isActive={pageFlag === 1}
+            >
+              {t('Per pipeline')}
+            </Button>
```

Comment:
Use `ToggleGroup` https://patternfly-react.surge.sh/components/toggle-group#default-with-single-selectable component from pf instead of 2 buttons.